src/bcrypt/__init__.py

```diff
@@ -76,3 +76,30 @@ def hashpw(password, salt):
         raise ValueError("Invalid salt")
 
     return _bcrypt.ffi.string(hashed)
+
+
+def kdf(password, salt, desired_key_bytes, rounds):
+    if isinstance(password, six.text_type) or isinstance(salt, six.text_type):
+        raise TypeError("Unicode-objects must be encoded before hashing")
+
+    if len(password) == 0 or len(salt) == 0:
+        raise ValueError("password and salt must not be empty")
+
+    if desired_key_bytes <= 0 or desired_key_bytes > 512:
+        raise ValueError("desired_key_bytes must be 1-512")
+
+    if rounds < 1:
```

Comment:
Function def for the underlying C is:

`bcrypt_pbkdf(const char *pass, size_t passlen, const uint8_t *salt, size_t saltlen, uint8_t *key, size_t keylen, unsigned int rounds)`

So it calls it rounds there too (plus py-bcrypt calls it rounds and for kwarg API compatibility we have to match that anyway. That's why it's called `desired_key_bytes` even though I don't like that variable name at all.)